core/src/main/java/org/apache/spark/shuffle/unsafe/UnsafeShuffleWriter.java

```diff
@@ -101,28 +99,25 @@
 
   public UnsafeShuffleWriter(
```

Comment:
The changes in this file are just to demonstrate how BinaryShuffleWriter might be used by non-core shuffle writers.